pkgs/tools/misc/supergfxctl/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, rustPlatform, fetchFromGitLab, pkg-config, udev, kmod }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "supergfxctl";
+  version = "2.0.5";
```

Comment:
Bump to latest release [4.0.4](https://gitlab.com/asus-linux/supergfxctl/-/releases#4.0.4)?

Reply:
I'd rather get this merged first before I keep updating this PR over and over again to be perfectly honest

Reply:
Yeah, let's just get this merged for now, it's gotten a bit ridiculous.
I won't be able to test this until the weekend, though.